rewards/utils/tree_utils.py

```diff
@@ -37,14 +37,13 @@ def calc_next_cycle_range(
 
     last_claim_end = tree_manager.last_publish_end_block()
     start_block = last_claim_end + 1
-
-    # Claim at last synced block
-    end_block = last_synced_block(chain)
-    if chain == Network.Arbitrum:
-        end_block = end_block - 100
+    synced_block = last_synced_block(chain)
+    metadata = get_claimable_metadata(chain, synced_block)
+    end_block = metadata["endBlock"]
+    assert end_block <= synced_block
 
     # Sanity check: Ensure start block is not too far in the past
-    assert start_block < end_block
+    # assert start_block < end_block
```

Comment:
If we're removing this sanity check, please remove the commented code.

Reply:
I readded this just now, we need pause cycles a bit before we mergethis changeand let the unclaimed snapshots run so that this doesn't cause a problem when we restart the cycles. The problem is that the cycles are ahead of the of the snapshots but won't be when we start unclaimed cycles consecutively.